src/libs/banners/banners.ts

```diff
@@ -312,3 +313,32 @@ export const getKeySyncBanner = (addr: string, email: string, keys: string[]) =>
   }
   return banner
 }
+
+export const getFirstCashbackBanners = ({
+  selectedAccountAddr,
+  firstCashbackConfettiStatusByAccount
+}: {
+  selectedAccountAddr: string
+  firstCashbackConfettiStatusByAccount: FirstCashbackConfettiStatusByAccount
+}): Banner[] => {
+  const banners: Banner[] = []
+
+  const shouldShowConfettiBanner = firstCashbackConfettiStatusByAccount[selectedAccountAddr]
```

Comment:
Maybe `shouldShowBanner` only, since it's in the context of `getFirstCashbackBanners`